### ISSUE TITLE
dev-env and init-clients for multiple chains

### ISSUE THREAD

**Issue by ancazamfir** [informalsystems/ibc-rs#619]
## Crate
n/a

## Summary
Currently the `dev-env` and `init-clients` scripts work for two chains. 
We should modify them, or write new ones such that the setup is done for all chains included in the config file. Also, the config file parameter should be optional with default `~/.hermes/config.toml`.
For example: `./dev-env` would start nodes for all chains specified in `~/.hermes/config.toml`. It would then perform light client initialization for all chains.

## Proposal

____

#### For Admin Use

- [x] Not duplicate issue
- [x] Appropriate labels applied
- [x] Appropriate milestone (priority) applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

**Comment by romac** [comment]
I would suggest we rewrite those scripts in Python as they are getting more complex and bash scripting is such a pain. What do you think?

**Comment by ancazamfir** [comment]
Agreed! btw, we should also add the config default to `run.py`. Not sure about others but I run `./e2e/run.py` often to make sure I don't break the clis.

**Comment by adizere** [comment]
This issue can be understood as a sub-issue of #797. The work in 797 should support more than two chains.

**Comment by romac** [comment]
Now that we have `gm` (and soon `gm-connector`) we do not those scripts anymore, so let's close this issue.
